app/services/transaction_search/date_range_searcher.rb

```diff
@@ -8,7 +8,11 @@ class DateRangeSearcher < BaseSearcher
     attr_reader :date_range_field
 
     def self.options
-      FIELDS.map { |field| [I18n.t(field, scope: "admin.transaction_search.date_range_fields"), field] }
+      if @billing_context
```

Comment:
I'm not a huge fan of this option. I get that "fulfilled_at" doesn't really make sense in the New/In Process context, but that doesn't really bother me that much if the options are there. And I think problem orders have a fulfilled_at, so "billing context" is not necessarily the right name for the option.

If we really wanted to make this nicer, I would see if we can find a way to pass those options more explicitly. Something more like `allowed_date_fields: [:ordered_at]`. I'm not quite sure how that would work, though. Could do a `select` from that option in the view. `TransactionSearch::DateRangeSearcher.options.select { |label, value| value.in?(@search_form.allowed_date_fields) }`

These options are just me throwing out ideas off the cuff; not sure if they're great ones or not.

Reply:
Okay. I mean, if you're not bothered by the extra options being there, I guess it's not a huge deal for me either as long as NU doesn't mind.